### ISSUE TITLE
Dependent/filtered relation/field aka parent & child relationship/field

### ISSUE THREAD

**Issue by derrickmehaffy** [strapi/strapi#8708]
### Please describe your feature request

- [ ] **I have created my request on the Product Board before I submitted this issue**
- [x] **I have looked at all the other requests on the Product Board before I submitted this issue**

### Summary

This feature request comes from various community requests (Forum, meetups, Slack, ect) and I will do my best to link to those to hopefully make the request clear.

The easiest example to explain is laid out like this, we have the following models:

- Make
- Model
- Sales

In this context we are talking about cars, so the `make` would contain a list of car manufacturers (Subaru, Toyota, Ford, Chevy, ect). Thus the `model` would contain some options like WRX, GT86, Mustang, Corvette.

The way we would setup the `sales` model would be a oneWay relation to each make/model, and in the context of this request when we create a new "sale" we would select the make (say subaru) now when I go to select the model I should only be able to see/select the "WRX" model since it's made by Subaru.

I should not be able to select "Toyota" "Mustang" as that car doesn't exist.

### Why is it needed?

Simply put, requested by our users

### Suggested solution(s)

Discussed in various places, effectively there needs to be a link between the two models and a filter pre-applied in the admin where the 1st option (parent) is included as a filter in the 2nd option (child).

### Related issue(s)/PR(s)

https://forum.strapi.io/t/customize-relations/1260
https://forum.strapi.io/t/limiting-relational-options-in-admin/1173
https://forum.strapi.io/t/get-all-model-in-custom-field/795
(There are more, I'll add them when I find them)

**Comment by derrickmehaffy** [comment]
This issue has been mentioned on **Strapi Community Forum**. There might be relevant details there:

https://forum.strapi.io/t/parent-child-relation/2434/2

**Comment by derrickmehaffy** [comment]
This issue has been mentioned on **Strapi Community Forum**. There might be relevant details there:

https://forum.strapi.io/t/user-research-share-how-you-use-plugins-in-strapi/3595/8

**Comment by tufail** [comment]
> This issue has been mentioned on **Strapi Community Forum**. There might be relevant details there:
> 
> https://forum.strapi.io/t/user-research-share-how-you-use-plugins-in-strapi/3595/8

Hi @derrickmehaffy, Thank you for adding this feature request.

**Comment by derrickmehaffy** [comment]
This issue has been mentioned on **Strapi Community Forum**. There might be relevant details there:

https://forum.strapi.io/t/dynamic-content-after-selection/11145/5

**Comment by glancer000** [comment]
> I neeeeed.

+1

**Comment by derrickmehaffy** [comment]
Hey all, we are currently moving many feature requests to our new feedback and feature request website to help clean up the GitHub issues and make it easier for us to review bug reports and fix them.

I have moved this feature to the following URL: https://feedback.strapi.io/customization/p/parent-child-relationshipfield
If you are interested in this feature please feel free to go there and add more comments and/or upvote it there.

For now I will close this and lock it so that all new information goes into our new feedback website.
Thanks!
